src/api/local-reporting-config-response.json

```diff
@@ -1,246 +1,203 @@
 {
-	"WordPress.com": {
+	"Product": {
```

Comment:
You can ignore this change! I just changed the local JSON file to be more self-descriptive, rather than making it have Automattic-centric terms. This makes seeing what is happening in the app clearer when running and debugging locally.